estate/models/estate_property.py

```diff
@@ -0,0 +1,32 @@
+from odoo import fields, models
+from dateutil.relativedelta import relativedelta
+
+
+class estate_property(models.Model):
+    _name="estate.property"
+    _description="Estate Model"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode=fields.Char()
+    date_availability = fields.Date(
+        copy=False,
+        default= (fields.Date.today() + relativedelta(months=3))
+        # default=fields.Date.add(fields.Date.today() , months=3)
+    )    
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True,copy=False)
+    bedrooms = fields.Integer(defualt=2 )
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
```

Comment:
```suggestion
    is_garage = fields.Boolean()
    is_garden = fields.Boolean()
```
Update this fields name to make it more effective to use in code